src/sentry/testutils/helpers/backups.py

```diff
@@ -107,9 +107,9 @@ def export_to_file(path: Path, scope: ExportScope) -> JSONData:
         if scope == ExportScope.Global:
```

Comment:
It feels a bit cleaner to me to abstract the logic for the scopes during the export process in a single function, which seems possible? That's also changing how it was previously done, so going to leave this as optional